runtime/src/accounts_db.rs

```diff
@@ -1255,10 +1317,57 @@ impl AccountsDB {
             .filter(|slot| !accounts_index.is_root(**slot))
             .collect();
         drop(accounts_index);
+        let mut storage_lock_elapsed = Measure::start("storage_lock_elapsed");
         let mut storage = self.storage.write().unwrap();
+        storage_lock_elapsed.stop();
+
+        let mut all_removed_slot_storages = vec![];
+        let mut total_removed_storage_entries = 0;
+        let mut total_removed_bytes = 0;
+
+        let mut remove_storages_elapsed = Measure::start("remove_storages_elapsed");
         for slot in non_roots {
-            storage.0.remove(&slot);
+            let slot_removed_storages = storage.0.remove(&slot);
+            total_removed_storage_entries +=
```

Comment:
nits: `if let Some(slot_removed_storages) = ...` would make the code a bit cleaner, perhaps?

Reply:
yup definitely, fixed!